common/djangoapps/track/backends/tests/test_oli.py

```diff
@@ -0,0 +1,201 @@
+"""
+Tests oli.py
+"""
+from __future__ import absolute_import
+
+from mock import patch
+import logging
+from django.test import TestCase
+from south.utils import datetime_utils as datetime
+from track.backends.oli import OLIAnalyticsBackend
+from student.tests.factories import UserFactory
+log = logging.getLogger(__name__)
```

Comment:
Should there be a newline between this assignment and the above import stanza(s)?